src/nxdoc/nuxeo-add-ons/nuxeo-enhanced-viewer/nuxeo-enhanced-viewer-installation.md

```diff
@@ -0,0 +1,254 @@
+---
+title: 'Installation and Configuration'
+description: 'Install and configure Nuxeo Enhanced Viewer'
+review:
+    comment: ''
+    date: '2022-07-01'
+    status: ok
+labels:
+    - addon
+toc: true
+tree_item_index: 50
+---
+
+## Presentation
+
+The Nuxeo Enhanced Viewer is provided by the integration of the ARender software which is to deploy separately from Nuxeo, and the [Nuxeo Enhanced Viewer Addon](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-arender).
+
+{{#> callout type='warning' heading='Private packages'}}
+You should contact your Nuxeo Administrator or Nuxeo sales representative to get access to these packages.
+{{/callout}}
+
+The ARender software is extended by Nuxeo to integrate the Nuxeo Connector and configurations/customizations.
+The Nuxeo Connector adds to the Previewer the capability to communicate with the Nuxeo REST API, and mostly leverages the [comment & annotation endpoints]({{page page='comments#rest-api'}}).
+
+The ARender software is provided through 5 Docker images to pull from our Docker registry docker-arender.packages.nuxeo.com:
+
+- nuxeo/arender-document-converter
+- nuxeo/arender-document-renderer
+- nuxeo/arender-document-service-broker
+- nuxeo/arender-document-text-handler
+- nuxeo/arender-ui
+
+NEV Architecture and communication between services:
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/Nuxeo Enhanced Viewer/NEV - Architecture 2.x
+    name: NEV - Architecture 2.x.png
+    addins#diagram#up_to_date
+--}}
+![NEV - Architecture 2.x](nx_asset://2370d315-4a7d-4e06-a25b-1f30085f55fd)
+
+The user is authenticated in ARender with an [OAuth2 Authorization Grant Flow]({{page page='using-oauth2#oauth-2-authorization-grant-flow'}}) against Nuxeo.
+
+{{#> callout type='warning' heading='deployment'}}
+In order to make the integration work on modern browsers, Nuxeo and NEV must be deployed under sibling domains, for instance `nuxeo.domain.org` and `nev.domain.org`. This is due to the OAuth2 challenge which occurs in an iframe, as the ARender UI is integrated in Nuxeo Web UI within an iframe.
+{{/callout}}
+
+## Nuxeo Configuration
+
+You need to configure the ARender Previewer URL, and an OAuth2 client in `nuxeo.conf`:
+
+```properties
+arender.server.previewer.host=https://arender-previewer-url
+nuxeo.arender.oauth2.client.create=true
+nuxeo.arender.oauth2.client.id=arender
+nuxeo.arender.oauth2.client.secret=OAUTH2_SECRET
+nuxeo.arender.oauth2.client.redirectURI=/login/oauth2/code/nuxeo
+```
+
+The `OAUTH2_SECRET` has to be replaced, there's no constraint on it.
+
+## ARender Configuration
+
+The preferred way to run NEV services is to use Kubernetes and Helm 3 as they provide interesting features such as auto-scaling and high availability.
+
+### Kubernetes
+
+The Helm Chart and installation instructions are available in the [ARender Documentation](https://docs.arender.io/fr/install/kubernetes/).
```

Comment:
Should rather link to the english install doc: https://docs.arender.io/install/kubernetes/